applications/plugins/CImgPlugin/CMakeLists.txt

```diff
@@ -86,7 +87,7 @@ add_library(${PROJECT_NAME} SHARED ${HEADER_FILES} ${SOURCE_FILES})
 
 target_compile_options(${PROJECT_NAME} PUBLIC ${CIMG_CFLAGS})
 target_compile_options(${PROJECT_NAME} PRIVATE "-DSOFA_BUILD_CIMGPLUGIN")
-target_link_libraries(${PROJECT_NAME} SofaCore ${EXTERNAL_LIBS})
+target_link_libraries(${PROJECT_NAME} SofaCore SofaSimulationCommon ${EXTERNAL_LIBS})

```

Comment:
SofaSimulationCommon  is needed?

Reply:
it is not, forgot how it ended there.
I removed it, along with the commented includes and pushed the updated code to remote.

Reply:
I didn't change the files/code/algos, I just moved classes around.